### ISSUE TITLE
Failures possibly related to Maria DB version

### ISSUE THREAD

**Issue by supersoju** [WordPress/phpunit-test-runner#150]
We've been seeing some consistent failures against our environment. A few examples are below:

```
Tests_dbDelta::test_column_type_change
Failed asserting that two arrays are identical.
--- Expected
+++ Actual
@@ @@
 Array &0 (
-    'wptests_dbdelta_test.id' => 'Changed type of wptests_dbdelta_test.id from bigint to int(11)'
+    'wptests_dbdelta_test.id' => 'Changed type of wptests_dbdelta_test.id from bigint(20) to int(11)'
 )

Tests_dbDelta::test_column_added
Failed asserting that two arrays are identical.
--- Expected
+++ Actual
@@ @@
 Array &0 (
+    'wptests_dbdelta_test.id' => 'Changed type of wptests_dbdelta_test.id from bigint(20) to bigint'
     'wptests_dbdelta_test.extra_col' => 'Added column wptests_dbdelta_test.extra_col'
 )
```

These all look to be unexpected MySQL responses, and our version of MariaDB is unique among other reporters.

Latest full report, with all failures, is available here: https://make.wordpress.org/hosting/test-results/r51510/a2hostinbot-r51510/

**Comment by supersoju** [comment]
We observed that other hosts using MariaDB with versions similar to ours were not having the issue, but when I opened the ticket there were no other hosts with the exact version that we had and maybe there were some clues in there.

**Comment by SergeyBiryukov** [comment]
Thanks for the report!

This appears to be an issue with WordPress core rather than the test runner.

It should now be fixed for WordPress 6.1 in [r53897](https://core.trac.wordpress.org/changeset/53897) / [#49364](https://core.trac.wordpress.org/ticket/49364).
